### ISSUE TITLE
Accessing the Algolia API full response

### ISSUE THREAD

**Issue by francescobm** [algolia/vue-instantsearch#316]
Hi, 

I am using vue-instantsearch for my ecommerce and have to say it's very handy and intuitive. 
However, I can't find a way to access the full response from the API.

If I check the network the endpoint is returning some properties other than the hits object. However, seems that the implementation like below is only allowing to access the hits object.

```vue
<template>
  <ais-index 
    :app-id="algoliaConfig.id" 
    :api-key="algoliaConfig.key" 
    :index-name="algoliaConfig.index" 
    :query-parameters="{hitsPerPage:33}">
  <ais-results>
       loop over results
   </ais-results>
  </ais-index>
</template>

<script>
  // here I need to be able to see the full API response 
  // so that I can access the other properties
  export default {
    data () {}
  }
</script>
```

**Comment by Haroenv** [comment]
which other properties do you need access to?

**Comment by rayrutjes** [comment]
I think we can expose the last response from the store so that you can deal with it as you wish.

You would access it like so:
`store.lastResponse`

WDYT @francescobm ?

**Comment by francescobm** [comment]
That would be very appreciated,
 
I assume it will take a bit of time before the release this feature, so for the time being I'll have to renounce to the vue-instantsearch components and I'll manually do a POST request to retrieve hits and facet_stats.

Thanks a lot for your work !

**Comment by Haroenv** [comment]
In the meantime you can use `store._helper.lastResponse`

**Comment by francescobm** [comment]
I've been able to find it in 
`this.searchStore._helper.lastResults._rawResults[0].facets_stats`

**Comment by Haroenv** [comment]
Thanks for sharing your workaround @francescobm, we'll expose this simpler soon, it's also useful for query rules ( #306 )

**Comment by francescobm** [comment]
Full example here, forget to mention that of course lastResults is asynch, so needs to be watched in order to be accessed. Would be nice to be able to have a promise available within ais-index

```
<template>
  <div>
    {{ priceStats.min }}
    {{ priceStats.max }}
  </div>
</template>

<script>
  import { Component } from 'vue-instantsearch'
  export default {
    mixins: [Component],
    data () {
      return {
        priceStats: {
          min: null,
          max: null
        }
      }
    },
    watch: {
      'searchStore._helper.lastResults': function (newVal, oldVal) {
        Object.assign(this.priceStats, newVal._rawResults[0].facets_stats.price)
      }
    }
  }
</script>
```

**Comment by rayrutjes** [comment]
> Would be nice to have a promise accessible from the custom component or the component where I declare ais-index

@francescobm not sure I understand what you mean, could you adapt your example with your ideal usage implementation?

**Comment by Haroenv** [comment]
this is addressed in v2 #416 (more specifically #505)
